src/utils/urlPaths.ts

```diff
@@ -23,11 +23,16 @@ export enum UrlPathTemplate {
   Logout = '/logout',
   MyPage = '/my-page',
   MyPageMessages = '/my-page/messages',
+  MyPageMyMessages = '/my-page/messages/my-messages',
   MyPageMyProfile = '/my-page/my-profile',
```

Comment:
Kan gjerne bare ta bort disse med en gang tror jeg
```suggestion
  MyPageMyMessages = '/my-page/messages/my-messages',
```

Reply:
Gjelder flere her også det. `MyPage`, `MyPageRegistrations`, etc. Får sjekke litt hva som evt må beholdes og ikke da

Reply:
evt kan det kanskje gjøres i en egen PR også. your call

Reply:
Tar det i egen PR når alle menyene er på plass tenker jeg